assets/javascript/apps/pipeline/stores/pipelineManagerStore.ts

```diff
@@ -50,6 +50,7 @@ const usePipelineManagerStore = create<PipelineManagerStoreType>((set, get) => (
           if (updatedFlow) {
             set({currentPipeline: pipeline});
             resolve();
+            // @ts-expect-error. This module is available
```

Comment:
I think you could update the `eslint` config to add `alertify` as a global.

Reply:
Cool. I'll do so in a new PR. Thanks for the tips

Reply:
Turns out it was something different. I'll PR a change.